Jenkinsfile

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env groovy
 common {
+  slackChannel = '#connect-eng'
+  upstreamProjects = 'confluentinc/schema-registry'
```

Comment:
How is this repo dependent upon SR?

Reply:
It has schema registry integration for Avro data being read from Kafka. https://github.com/confluentinc/camus/blob/confluent-master/camus-kafka-coders/pom.xml#L17